doc/performance_blog_post/performance_blog_post.md

```diff
@@ -0,0 +1,120 @@
+# How we Improved the Files Virtual Schema Performance
+
+Our [Virtual Schemas for document files](https://github.com/exasol/virtual-schema-common-document-files) allow our customer to integrate formatted data from files into their Exasol database. Our customers use this feature for integrating external data on demand into their analytics calculation and for building Extract, Transform and Load (ETL) pipelines.
+
+For Exasol speed matters. So does it for our integrations. For customers it's crucial that the queries that include external data run as fast as possible. In this blog post I want to give you some insights from our recent performance improvements of the Virtual Schema for [Apache Parquet](https://parquet.apache.org/) files.
+
+## How it Begins
+
+For me this project started when our product manager asked me to improve the performance of the Parquet file loading. He compared the performance of our products with other solutions. As a result of this comparison, we realized that our Parquet loading needs major improvements.
+
+In order to do professional performance improvements we use the simple workflow:
+
+Measure &#8594; Improve &#8594; Measure.
+
+In my eyes, this simple strategy brings lots of advantages:
+
+First, it allows you to check whether a change helped or if it even made some functionality slower. For example improving the performance of loading big Parquet files can easily slow down the loading of many small files. Second, this strategy allows you to define a goal when to stop. Otherwise, you will keep improving forever since there's always a space for improvements.
```

Comment:
```suggestion
First, it allows you to check whether a change helped or if it even made some functionality slower. For example, improving the performance of loading big Parquet files can easily slow down the loading of many small files. Second, this strategy allows you to define a goal when to stop. Otherwise, you will keep improving forever since there's always a space for improvements.
```